src/sentry/incidents/logic.py

```diff
@@ -1385,12 +1400,13 @@ def create_alert_rule_trigger_action(
         alert_rule_trigger=trigger,
         type=type.value,
         target_type=target_type.value,
-        target_identifier=target_identifier,
-        target_display=target_display,
+        target_identifier=target.identifier,
+        target_display=target.display,
         integration_id=integration_id,
         sentry_app_id=sentry_app_id,
         sentry_app_config=sentry_app_config,
-    )
+    )  # type: ignore[misc]
+    # Ignore assigning sentry_app_config (dict[str, Any]) as JSONField
```

Comment:
this means that the annotation for JSONField is incorrect -- there should be a bunch of examples for this throughout the codebase (tl;dr it needs a manual `Field` annotation because our JSONField is a huge hack!)

Reply:
Done: https://github.com/getsentry/sentry/pull/75541/commits/677791c53200e26fccc5acb4226bf76a15a2200e